--- a/kittentts/onnx_model.py
+++ b/kittentts/onnx_model.py
@@ -1,4 +1,3 @@
-from misaki import en, espeak
 import numpy as np
 import phonemizer
 import soundfile as sf
--- a/kittentts/__init__.py
+++ b/kittentts/__init__.py
@@ -1 +1 @@
-from kittentts.get_model import get_model, KittenTTS
+from kittentts.onnx_model import KittenTTS_1_Onnx
@@ -3 +3 @@
-__version__ = "0.1.0"
+__version__ = "0.8.1"
@@ -7 +7 @@
-__all__ = ["get_model", "KittenTTS"]
+__all__ = ["KittenTTS_1_Onnx"]
