--- a/kittentts/onnx_model.py
+++ b/kittentts/onnx_model.py
@@ -1 +0,0 @@
-from misaki import en, espeak
@@ -80 +79 @@
-    def __init__(self, model_path="kitten_tts_nano_preview.onnx", voices_path="voices.npz", speed_priors={}, voice_aliases={}):
+    def __init__(self, model_path="kitten_tts_nano_preview.onnx", voices_path="voices.npz", speed_priors={}, voice_aliases={}, session_options=None, providers=None):
@@ -89 +88,5 @@
-        self.session = ort.InferenceSession(model_path)
+        self.session = ort.InferenceSession(
+            model_path,
+            sess_options=session_options,
+            providers=providers or ["CPUExecutionProvider"],
+        )
--- a/kittentts/__init__.py
+++ b/kittentts/__init__.py
@@ -1 +1 @@
-from kittentts.get_model import get_model, KittenTTS
+from kittentts.onnx_model import KittenTTS_1_Onnx
@@ -3 +3 @@
-__version__ = "0.1.0"
+__version__ = "0.8.1"
@@ -7 +7 @@
-__all__ = ["get_model", "KittenTTS"]
+__all__ = ["KittenTTS_1_Onnx"]
